server/api/user_handler.py

```diff
@@ -35,13 +40,21 @@ def register():
     token = jwt.encode({"user": username, "exp": datetime.utcnow() + timedelta(minutes=exp)}, \
                        app.app.config['JWT_SECRET'])
 
-    return jsonify({"auth_token": token}), 201
+    response = jsonify({"message": "success"})
+    response.status_code = 201
+    response.set_cookie('auth_token', value=token)
```

Comment:
```suggestion
    response.set_cookie('auth_token', value=token, httponly=True)
```

should set `httponly` so that the cookies not accessible through javascript